content/pt-br/docs/reference/glossary/pod-disruption.md

```diff
@@ -0,0 +1,21 @@
+---
+id: pod-disruption
+title: Interrupção do Pod
+full_link: /docs/concepts/workloads/pods/disruptions/
+date: 2021-05-12
+short_description: >
+  O processo pelo qual Pods ou Nós são interrompidos de forma voluntária ou involuntária.
+
+aka:
+related:
+  - pod
+  - container
+tags:
+  - operation
+---
+
+[Interrupção do Pod](/docs/concepts/workloads/pods/disruptions/) é o processo pelo qual Pods ou Nós são interrompidos de forma voluntária ou involuntária.
+
+<!--more-->
+
+Interrupções voluntárias são iniciadas intencionalmente pelos donos das aplicações ou administradores dos clusters. Interrupções involuntárias não são intencionais e podem ser encadeadas por problemas inevitáveis como Nós com poucos recursos, ou por exclusões acidentais.
```

Comment:
How do you feel about replacing "interrupção" with "disrupção" in all occurrences of this page?

Not mandatory, but wanted to gauge how strongly you feel about "interrupção". It feels to me that "disrupção" would be a more faithful translation.

Reply:
Done!